spec/features/idv/end_to_end_idv_spec.rb

```diff
@@ -340,11 +340,15 @@ def validate_personal_key_page
     expect(page).to have_content(t('forms.personal_key_partial.acknowledgement.text'))
     expect(page).to have_content(t('forms.personal_key_partial.acknowledgement.help_link_text'))
     expect(page).to have_content(t('idv.messages.confirm'))
-    expect_step_indicator_current_step(t('step_indicator.flows.idv.secure_account'))
     expect(page).to have_css(
       '.step-indicator__step--complete',
       text: t('step_indicator.flows.idv.verify_phone_or_address'),
     )
+    expect(page).to have_css(
+      '.step-indicator__step--complete',
+      text: t('step_indicator.flows.idv.secure_account'),
+    )
+    expect(page).not_to have_css('.step-indicator__step--current', wait: 5)
```

Comment:
Do we need the `wait` here?

```suggestion
    expect(page).not_to have_css('.step-indicator__step--current')
```